src/main/java/frc/robot/commands/LimelightAlign.java

```diff
@@ -31,15 +31,14 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    SmartDashboard.putNumber("exec", 1);
     double adjustment = m_LimelightSubsystem.calculatePID();
-    m_drivebaseSubsystem.setSpeeds(adjustment, adjustment * -1);
+    m_drivebaseSubsystem.arcadeDrive(adjustment);
   }
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
-    m_drivebaseSubsystem.setSpeeds(0, 0);
+    m_drivebaseSubsystem.arcadeDrive(0);
```

Comment:
Is this needed?  if the command is interrupted, should just stay at the heading you are at.

Reply:
To help make sure the limelight doesn't accidentally keep turning the robot, probably not needed but it shouldn't affect anything.

Reply:
I'll change it, though